css/style.css

```diff
@@ -1,3 +1,61 @@
+body {
+  width: 600px;
+  background: lightgrey;
+  margin: auto;
+}
+
+main{
+  text-align: center;
+}
+
 h1 {
-  color: salmon;
+  color: #4d4d4d;
+  font-size: 20px;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+h2 {
+  color: #4d4d4d;
+  font-size: 10px;
+  font-family: sans-serif;
+  font-weight: normal;
+}
+
+button {
+  color: #4d4d4d;
+  font-size: 10px;
+  background-color: white;
+  padding: 5px 30px;
+  border: 1px solid #3d3d3d;
+  margin: 10px;
+  margin-bottom: 25px;
+}
+
+.flex-container{
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is applied to all flex containers by default, so you don't have to write it out unless you're overriding an already existing `flex-direction` property.